etc/generate-envoy-config/envoy.jsonnet

```diff
@@ -0,0 +1,196 @@
+// Services that we want to proxy; where they are and how to route them.
+local services = {
+  'pachd-grpc': {
+    internal_port: 1650,
+    external_port: 30650,
+    grpc: true,
+    service: 'pachd-proxy-backend',
+    routes: [
+      {
+        match: {
+          grpc: {},
+          prefix: '/',
+        },
+        route: {
+          cluster: 'pachd-grpc',
+          timeout: '604800s',
+        },
+      },
+    ],
+    health_check: {
+      grpc_health_check: {},
+      healthy_threshold: 1,
+      interval: '10s',
+      timeout: '10s',
+      unhealthy_threshold: 2,
+      no_traffic_interval: '10s',
+      no_traffic_healthy_interval: '10s',
+    },
+  },
+  'pachd-s3': {
+    internal_port: 1600,
```

Comment:
why are both internal ports and external ports required?

Reply:
We don't use the external port.  I kept them as documentation.

Reply:
gotcha makes sense. Could you leave a comment about that somewhere? I feel like that would confuse me if I stumbled across it otherwise